data/products/sap/hardened/sle15/config.yaml

```diff
@@ -1,4 +1,9 @@
 config:
+  files:
+    sles4sap-variant:
```

Comment:
Not a blocker but cosmetic: I'd replace "sles4sap-variant" with "sles-sap-hardened-variant-id" to be consistent with the "sles-hardened-variant-id" name.

Reply:
This overwrites the `sles4sap-variant` namespace from `data/products/sap/sle15/config.yaml`. We could null it instead and add a namespace `sles-sap-hardened-varian-id` if you prefer. I did that in other places where the resulting comment in config.kiwi seemed inappropriate otherwise but I didn't think it mattered in this case.